capplets/desktop-icons/mate-di-capplet.c

```diff
@@ -0,0 +1,380 @@
+/*
+ *  Authors: Luca Cavalli <loopback@slackit.org>
+ *
+ *  Copyright 2005-2006 Luca Cavalli
+ *  Copyright 2008 Thomas Wood <thos@gnome.org>
+ *  Copyright 2010 Perberos <perberos@gmail.com>
+ *  Copyright 2019 Laurent Napias
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of version 2 of the GNU General Public License
+ *  as published by the Free Software Foundation
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Street #330, Boston, MA 02110-1301, USA.
```

Comment:
Update [FSF address](https://www.fsf.org/about/contact/): 
```
Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Reply:
Copy/pate from https://github.com/tamplan/mate-control-center/blob/f741389053ce946aca1baa227c6f2fb011469d72/capplets/default-applications/mate-da-capplet.c#L19

Done in my files, preparing new PR for 4/5 files with wrong FSF address...